content/deprecated-components/markdown-editor.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Markdown editor
```

Comment:
This has been moved to Primer Recipes and doesn't need to ported over https://ui.githubapp.com/storybook/?path=/docs/recipes-markdowneditor--readme

Reply:
Is it possible someone is using this component in dotcom and needs to reference the deprecated docs?

Reply:
Not seeing it in [Primer Query](https://musical-adventure-wlr3n3k.pages.github.io/?name=markdowneditor)